### PR TITLE
Bump: arista.cvp Ansible collection requirement to latest version

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Bump Ansible collection requirements:
  - Minimum `arista.cvp` version 3.11.0.

### CODE REVIEW

File: ansible_collections/arista/avd/docs/release-notes/5.x.x.md [hunk 1]
```diff
@@ -16,9 +16,14 @@ title: Release Notes for AVD 5.x.x
 
 ### Changes to requirements
 
-- AVD v.5.0.0 requires Python version 3.10 or newer.
-- AVD now requires the `cvprac` Python library to be version 1.4.0 or higher.<br>
-  The latest version can be installed with `pip install "cvprac>=1.4.0" --upgrade`. has been removed from the `arista.avd` Ansible collection in v5.0.0.
+- Python requirements:
+  - AVD v.5.0.0 requires Python version 3.10 or newer.
+  - AVD now requires the `cvprac` Python library to be version 1.4.0 or higher.<br>
+The latest version can be installed with `pip install "cvprac>=1.4.0" --upgrade`.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/docs/release-notes/5.x.x.md]:
```suggestion
  - AVD now requires the `cvprac` Python library to be version 1.4.0 or higher.<br>
    The latest version can be installed with `pip install "cvprac>=1.4.0" --upgrade`.
```

File: ansible_collections/arista/avd/galaxy.yml [hunk 2]
```diff
@@ -36,9 +36,9 @@ tags: ['arista', 'network', 'networking', 'eos', 'avd', 'cloudvision', 'cvp']
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
 dependencies:
-    "arista.cvp": ">=3.10.1"
-    "arista.eos": ">=7.0.0"
-    "ansible.utils": ">=3.0.0"
+    "arista.cvp": ">=3.11.0"
+    "arista.eos": ">=10.0.1"
+    "ansible.utils": ">=5.1.2"
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/galaxy.yml]: We need to scan these for python and ansible requirements. So we ensure we are in sync on all our reqs.

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/galaxy.yml]: Done, for ansible.utils we need to bump the python requirement to `netaddr>=0.10.1`

File: ansible_collections/arista/avd/galaxy.yml [hunk 3]
```diff
@@ -36,7 +36,7 @@ tags: ['arista', 'network', 'networking', 'eos', 'avd', 'cloudvision', 'cvp']
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
 dependencies:
-    "arista.cvp": ">=3.10.1"
+    "arista.cvp": ">=3.11.1"
```
Comment (carlbuchmann) [inline ansible_collections/arista/avd/galaxy.yml]:
```suggestion
    "arista.cvp": ">=3.11.0"
```

**Review by ClausHolbechArista — APPROVED** [review]

**Pull request merged**